source/type/pb_converter.cpp

```diff
@@ -1240,6 +1239,7 @@ void PbCreator::Filler::fill_messages(const VjStopTimes* vj_stoptimes,
     // here we add all the impacts on stop_point and stop_area of section.from
     const auto& from_sp = vj_stoptimes->stop_times.front()->stop_point;
     for (const auto& message: from_sp->get_applicable_messages(pb_creator.now, pb_creator.action_period)) {
+        if (message->is_only_line_section()) { continue; }
```

Comment:
so if the message contains a `line_section_impact` and a `stop_point_impact`, we find also `line_section_impact` in pt_display_info?

Reply:
Yeah, all of that work is only about choosing to display or not the disruption, not about custom render the disruption about the context

Reply:
ok